src/buildpack-registry.ts

```diff
@@ -189,7 +189,7 @@ export class BuildpackRegistry {
     }
 
     if (readme.content) {
-      data.readme = `\n${Buffer.from(readme.content, readme.encoding).toString()}`
+      data.readme = `\n${Buffer.from(readme.content, readme.encoding as BufferEncoding).toString()}`
```

Comment:
Not going to stop you here but, ideally, you should drop the cast and validate that `readme.encoding` is one of the allowed [BufferEncoding](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/buffer.d.ts#L246-L258) values and error if not.

Reply:
Yeah, I took a stab at that over here: https://github.com/heroku/buildpack-registry.js/pull/11/files#diff-49d1faefcce7f444c95c3e84e4bf75307b302177ebd0225f7a7e4311b1b33ec5R211-R215. I didn't love it -- it requires encoding all the known options into this library. AFAICT, there's no way to introspect what the allowable `BufferEncoding` strings are at runtime (except by hard coding them).